src/js/edu-benefits/containers/EduBenefitsApp.jsx

```diff
@@ -14,13 +14,13 @@ import PerfPanel from '../components/debug/PerfPanel';
 import RoutesDropdown from '../components/debug/RoutesDropdown';
 
 import { isValidSection } from '../utils/validations';
-import { ensureFieldsInitialized, updateCompletedStatus } from '../actions/index';
+import { ensureSectionInitialized, updateCompletedStatus } from '../actions/index';
```

Comment:
Just thinking about how the name `section` fits into our new naming convention (`chapter` > `page` > `block`). Is `section` meant to refer to all the questions on a `page` or can it sometimes be a subset of that?